howtos/add-new-repository.md

```diff
@@ -0,0 +1,13 @@
+
+# Add a new repository to Eiffel Community
+
+Want to extend the Eiffel community with another project?
+
+1. [Contact the community](../CONTACT.md) about the new repository you want to create, to see if it's a good fit.
+1. Read through the [Code of Conduct](https://github.com/eiffel-community/.github/blob/master/CODE_OF_CONDUCT.md) and [contribution guidelines](https://github.com/eiffel-community/.github/blob/master/CONTRIBUTING.md) for Eiffel Community and understand the content.
+
+1. Decide on a name for the repository. Naming conventions are to use "eiffel-" as prefix to the repository name.
+1. Create the repository in the [eiffel-community organization](https://github.com/organizations/eiffel-community/repositories/new) The repository should be public.
```

Comment:
Would we accept the option to 'import existing repositories' to the community as well, or should they always be created clean?

Reply:
Yes that was my intention also, perhaps I should explicitly state this.